README.md

```diff
@@ -306,3 +317,8 @@ Blogs:
 - [Building a Large-scale Distributed Storage System Based on Raft](https://www.cncf.io/blog/2019/11/04/building-a-large-scale-distributed-storage-system-based-on-raft/), by Ed Huang
 
 ## Benchmark & Testing
+
+## Build A Database from Scratch
+- [Let's Build a Simple Database](https://cstack.github.io/db_tutorial/)
+- [builddatabase](https://github.com/ngaut/builddatabase)
+- [Writing a sqlite clone from scratch in C](https://github.com/cstack/db_tutorial)
```

Comment:
this is the same with the first one "Let's Build a Simple Database"

Reply:
Yes sorry.